Core/src/Exception/ServiceException.php

```diff
@@ -57,13 +57,13 @@ class ServiceException extends GoogleException
     /**
      * Handle previous exceptions differently here.
      *
-     * @param string $message
+     * @param string|null $message
      * @param int $code
      * @param Exception|null $serviceException
      * @param array $metadata [optional] Exception metadata.
      */
     public function __construct(
-        $message,
+        $message = null,
```

Comment:
What necessitates this change?

Reply:
I tried to explain this in https://github.com/googleapis/google-cloud-php/commit/353a0b4523f832a1436da62c3a626908f1516df6

I don't know if the commit is still available when I force-push the suggested changes, so I'll copy the commit message here:

Allow `ServiceException` creation with null as the message

Some tests explicitly set the message to `null` which breaks the
creation of parent exceptions with newer PHP versions.

Setting a default `$message = ''` does not prevent the problem, so
we declare it with `null` being allowed, and check and replace it in the
`ServiceException` constructor.

The change is made in the `ServiceException` instead of the tests
calling it incorrectly to reduce the scope of updated files in the
context of the PR that handles the problem.